File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJStubBasedElement.java
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces;

import com.intellij.psi.PsiElement;
import com.intellij.psi.StubBasedPsiElement;
import com.intellij.psi.stubs.StubElement;

public interface ObjJStubBasedElement<StubT extends StubElement> extends  StubBasedPsiElement<StubT> {
}

File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJStubBasedElement.kt
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces

import com.intellij.psi.PsiElement
import com.intellij.psi.StubBasedPsiElement
import com.intellij.psi.stubs.StubElement

interface ObjJStubBasedElement<StubT:StubElement<*>> : StubBasedPsiElement<StubT>
